cnames_active.js

```diff
@@ -449,7 +449,7 @@ var cnames_active = {
   "casl": "stalniy.github.io/casl",
   "cassie": "milesgitgud.github.io/cassie",
   "catbox": "samsamson33.github.io/catbox.js",
-  "cath": "night0721.github.io/cath",
+  "cath": "cname.vercel-dns.com.",
```

Comment:
```suggestion
  "cath": "cname.vercel-dns.com", // noCF
```
Looks like you've got a trailing period in there that isn't needed, and when using Vercel you'll want the Cloudflare proxy disabled